### ISSUE TITLE
Rename the span_frames_min_duration property

### ISSUE THREAD

**Issue by danielfariati** [elastic/apm-agent-java#441]
The `span_frames_min_duration` seems to be used to decide whether or not to collect stack traces for a span.
The problem is that this information only becomes clear after reading the full documentation, as the name of the property does not make this easy to guess.
It would be better if the name of the property was clear enough for the user to know what it is used for.
Maybe `stack_trace_span_min_duration`, for example.

**Comment by felixbarny** [comment]
I agree this is a confusing name. The problem is that it has to be aligned with the other Elastic APM agents. But that doesn't mean we can't change it 🙂 

How do you like `span_stack_trace_min_duration`?

**Comment by danielfariati** [comment]
Much better! `span_stack_trace_min_duration` is a lot more clear.
